src/components/FavListings.js

```diff
@@ -61,41 +61,22 @@ const FavListings = (props) => {
               whiteSpace: "nowrap",
             }}
           >
-            <Carousel.Item activeIndex={indexC1} onSelect={handleSelect}>
+           {listing.images.map((image, index) => (
```

Comment:
@rohailramesh hey Ro, check this little refactor I did on the FavListings component. **Mapping over the images allows us to show all of them**, instead of just the three that were manually set before :)

In plain English it would be: f**or each image of the listing, lets create a carousel.item for it...**

Dont worry if its confusing tho!